src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -750,6 +758,14 @@ function build_eal_list() {
 	    'to allow connections from any IPv6 address.' . '<br/>' . 'Responder Only must be set and ' . 
 	    'Peer IP Address cannot be used for Remote Identifier.'), 'info', false),
 	    '</div>');
+	$group->add(new Form_Input(
+    'ikeport',
+		'Remote NAT-T port',
```

Comment:
It looks like this is used any time the GUI value is set, so it's not really a "NAT-T" port, but a "Remote IKE Port".